### ISSUE TITLE
Pathfinding: add support for goals made up of multiple nodes

### ISSUE THREAD

**Issue by davebaol** [libgdx/gdx-ai#24]
See discussion and proposed implementation here, https://github.com/libgdx/gdx-ai/issues/7#issuecomment-71741086

Ideally, this kind of goal could be specified by an explicit set of nodes or abstract properties.

**Comment by piotr-j** [comment]
Any progress on this issue? Would be quite useful.

**Comment by davebaol** [comment]
@piotr-j 
No, sorry. Rather busy these days. Hopefully I'll have the time in the near future.
PR welcome, of course.  :smile:

**Comment by piotr-j** [comment]
Ill look into it once I figure out simple working ai, perhaps next week.

**Comment by piotr-j** [comment]
Does arbitrary list of endNodes makes sense? Since there is no relation between, they can be on opposite sides of the graph and that would result in basically endNodes.size searches that could as well be separate searches. 

As abstract properties, something like any node within maximum cost from endNode would make some sense. That would probably require full search and then backtracking till we find node that exceeds the specified cost. 

Any thoughts on that?
